style.css

```diff
@@ -0,0 +1,107 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.main-container {
+  color: whitesmoke;
+  background-color: #1c1a19;
+  min-width: 375px;
+  width: 100%;
+}
+
+.logo {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  background: #3e3c3c;
+  padding-left: 24px;
+  padding-right: 24px;
+  height: 92px;
+  align-items: center;
+}
+
+.me {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  height: 22px;
+  width: 80px;
+  left: 142px;
+}
+
+.initial {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  flex-wrap: wrap;
+  min-height: 592px;
+  height: 100vh;
+  min-width: 375px;
+  width: 100%;
+  padding-left: 24px;
+  padding-right: 24px;
+  background: url("images/illustration-header-mobile.png");
+  background-repeat: no-repeat;
+  background-position: right bottom;
+}
+
+.information {
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  letter-spacing: 0.4px;
+  font-size: 58px;
+  line-height: 70px;
+  color: #f55800;
+}
```

Comment:
- [OPTIONAL] Kindly make sure to align correctly the main title in order to look like the provided Figma design. :wink: 

     - ### Your design
    ![image](https://user-images.githubusercontent.com/87197412/198530994-64c1404d-eac9-4cf2-b884-275cd5052204.png)

     - ### Figma design
     ![image](https://user-images.githubusercontent.com/87197412/198531076-0013a695-9ea3-43a0-a2b9-bf72b8158d74.png)

Reply:
Thank you, I didn't notice these. I will change them accordingly!!